### DISCUSSION TITLE
Suggestion/Bug button on in-game menu

### DISCUSSION THREAD

**Discussion by Blastxu** [voxeltycoon/issues#469] [UI&UX]
I think it would be easier to suggest things/report bugs if there was a dedicated button for it on little gear menu next to the settings. Also it would be great if that menu also had an option to take an in-game screenshot to send it with the bug report/suggestion.

**Comment by kiwinuken** [comment]
There's an interesting functionality implemented by the developer of a game called "Software Inc": not only the player can report a bug from inside the game itself, like you propose - assuming the bug doesn't crash the game, of course - by writing the report, to which a log is automatically attached, but the player can also immediately attach a screenshot to it as well. I think the thought behind it is that the easier it is for players to create a complete and meaningful report, the more willing they'll be to report bugs. I have used that system a couple times myself, it's very straightforward to use.

**Comment by Blastxu** [comment]
Yeah, I've also seen a similar system on The Tenants game, it automatically attaches a screenshot and you can also give a rating from 1 to 5 on how much the issue is bothering you.

**Comment by victorleblais** [comment]
The only issue with that kind of reporting is that people would not check if the issue/suggestion has already been reported, leading to a LOT of filtering/reading work to be done by the devs or volunteers. It's already hell with people not even bothering to type one word in the search bar to check so I'm not sure if this is a good idea.

Having to make an "effort" can lead to less time wasted for the devs. If you can't be bothered to spend 5 minutes online to type your thing, then maybe what you wanted to type wasn't worth it...
